scripts/src/commands/build-ios.sh

```
@@ -0,0 +1,132 @@
+#!/usr/bin/env bash
+
+device_default="iPhone 12"
+build_actions=(build)
+
+usage() {
+  echo "Build an iOS project stored in the 'ios' directory of a package."
+  echo "This must be run from the package directory."
+  echo ""
+  echo "USAGE: build-ios <options>"
+  echo ""
+  echo "OPTIONS:"
+  echo "  --clean                 Optional. Clean up intermediate files before building."
+  echo "  -d, --device <name>     Optional. Name of the simulated iOS device to target."
+  echo "                          Defaults to '${device_defaukt}'."
+  echo "  -s, --scheme <name>     Required. Name of the Xcode scheme to build"
+  echo "  -w, --workspace <name>  Required. Name of the Xcode workspace file (without extension)."
+  echo "                          This file must exist under the 'ios' directory."
+  echo "                          Example: 'MyApp' --> ios/MyApp.xcworkspace"
+  echo ""
+  exit 1
+}
+
+xcpretty=$(which xcpretty)
+if [[ -z "${xcpretty}" ]]; then
+  echo ""
+  echo "ERROR: xcpretty is required to run this script. Install it using 'gem install xcpretty'."
+  echo "       You may need to use 'sudo' to install it."
+  echo ""
+  exit 1
+fi
```

We should drop `xcpretty`. It hasn't been updated for some time and is known to swallow errors.

---

will do. you might want to update RNTA, too, since it uses xcpretty in its CI loops.

---

Where did you see it? I'm pretty sure I removed it all:
```
~rnta % git grep xcpretty
~rnta %
```